_students/sina_khalili/homework2/dags/homework2.py

```diff
@@ -0,0 +1,76 @@
+"""
+Dag file to get some astronaut data as JSON and transform it to csv
+"""
+import json
+import requests
+import csv
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.operators.python_operator import PythonOperator
+from airflow.operators.bash_operator import BashOperator
+
+request = "http://api.open-notify.org/astros.json"
+
+default_args = {
+    'owner': 'airflow',
+    'start_date': datetime(2019, 7, 21),
+    'depends_on_past': False,
+    'email': ['khalili@sfu.ca'],
+    'email_on_failure': False,
+    'email_on_retry': False,
+    'retries': 1,
+}
+dag = DAG(
+    'space_dag',
+    default_args=default_args,
+    description='gets astrodata',
+    schedule_interval=timedelta(days=10),
+)
+
+task1 = BashOperator(
+    task_id="blastoff",
+    bash_command="echo '🚀 blastoff'",
```

Comment:
The 🚀 causes an error for airflow to load the dag

Reply:
Lol removed 👍